app/js/controllers/redirection.js

```diff
@@ -1,42 +1,16 @@
 'use strict';
 
-angular.module('ddsApp').controller('RedirectionCtrl', function($scope, SituationService, $sce) {
+angular.module('ddsApp').controller('RedirectionCtrl', function($scope, $localStorage, $sessionStorage, $stateParams, SituationService) {
+  $sessionStorage.situation = $sessionStorage.situation || { _id: $localStorage.trampoline && $localStorage.trampoline.id };
```

Comment:
What is `trampoline`?

Reply:
C'est un passage dans le localStorage pour faire passer une donnée d'une session à une autre. Un #Hack